x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.test.tsx

```diff
@@ -333,8 +335,8 @@ describe('DetectionEnginePageComponent', () => {
   });
 
   it('the pageFiltersUpdateHandler updates status when a multi status filter is passed', async () => {
-    (FilterGroup as jest.Mock).mockImplementationOnce(
-      getMockedFilterGroupWithCustomFilters([
+    jest.spyOn(alertFilterControlsPackage, 'AlertFilterControls').mockImplementation(
```

Comment:
Should we use `mockImplementationOnce` because this will change implementation for all further tests.

Or we can define initial implementation in `beforeEach` instead of `beforeAll`?

Reply:
My bad, I actually meant to use `Once`